### PR TITLE
Fixed #184

### LINKED ISSUE #184: Browser back button doesn't direct to WikiContrib landing page
The browser back button, when clicked from "User's Activity" page doesn't direct to the WikiContrib home page. 

wikicontrib-screencast

### PR DESCRIPTION
**Pull request by shivanigangadharan**
I used the event handler 'onpopstate' to change the path history. Added code for the same in activity.js.
Let me know in case this doesn't work, thanks :)

### CODE REVIEW

**Comment by srish** [comment]
@sharvaniharan Hi! This PR is related to which issue?

File: frontend/WikiContrib-Frontend/src/components/activity.js [hunk 1]
```diff
@@ -7,6 +7,13 @@ import phabricatorPlatformIcon from '../img/phabricatorPlatformIcon.png';
 /**
  * Show all the user Commits on a specific day
  */
+window.history.pushState({page: 1}, "", "");
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/components/activity.js:10]: As per the pushState's syntax [here](https://developer.mozilla.org/en-US/docs/Web/API/History/pushState), I am wondering if this line should look like this `window.history.pushState({page: 1}, "WikiContrib", "/");` and include the title and url as well?

File: frontend/WikiContrib-Frontend/src/components/activity.js [hunk 2]
```diff
@@ -7,6 +7,13 @@ import phabricatorPlatformIcon from '../img/phabricatorPlatformIcon.png';
...
+
+window.onpopstate = function(event) {
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/components/activity.js:12]: So, if we include the new history url in the previous statement already, do we need to add this function which is doing no different?

Comment (shivanigangadharan) [reply frontend/WikiContrib-Frontend/src/components/activity.js:12]: I see, rightly said. Including only history.pushState does the job. Thank you for the suggestion :)
I have sent a PR after creating a new branch and making changes there as you mentioned. Hope that solves the issue.

Comment (shivanigangadharan) [reply frontend/WikiContrib-Frontend/src/components/activity.js:12]: Also, now that I have sent the new PR with only the required commit, I can close this PR right?

**Comment by rammanoj** [comment]
@shivanigangadharan I guess this and #188 refer to the same change ?

**Comment by shivanigangadharan** [comment]
> @shivanigangadharan I guess this and #188 refer to the same change ? Why did you create two PRs ?

Yes, both refer to the same change. Actually when I sent this PR, I had not created my own branch and it contained my previous commits as well. So @srish suggested that I should create a new PR from my branch. I suppose I can close this PR now since the new changes have been sent from the separate PR.

**Comment by shivanigangadharan** [comment]
@srish I've sent the new PR #188 with the new changes, so closing this one. Thank you :)

**Pull request closed without merging**